CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/editors/StringEditorFactory.java

```diff
@@ -80,40 +80,40 @@ protected Node createEditorControls() {
             controls.getColumnConstraints().add(cc);
             final RowConstraints rc = new RowConstraints();
             rc.setVgrow(Priority.ALWAYS);
-            controls.getRowConstraints().add(rc);
-
-            textArea = new TextArea();
-            textArea.setWrapText(true);
-            textArea.textProperty().addListener((o, n, v) -> update());
-            textArea.addEventFilter(KeyEvent.KEY_PRESSED, e -> {
+            controls.getRowConstraints().add(rc);            
+            
+            spellCheckingTextArea = new SpellCheckingTextArea(true);
+            spellCheckingTextArea.setWrapText(true);
+            spellCheckingTextArea.textProperty().addListener((o, n, v) -> update());
+            spellCheckingTextArea.addEventFilter(KeyEvent.KEY_PRESSED, e -> {
                 if (e.getCode() == KeyCode.DELETE) {
-                    IndexRange selection = textArea.getSelection();
+                    IndexRange selection = spellCheckingTextArea.getSelection();
                     if (selection.getLength() == 0) {
-                        textArea.deleteNextChar();
+                        spellCheckingTextArea.deleteNextChar();
                     } else {
-                        textArea.deleteText(selection);
+                        spellCheckingTextArea.deleteText(selection);
                     }
                     e.consume();
-                } else if (e.isShortcutDown() && e.isShiftDown() && (e.getCode() == KeyCode.RIGHT)) {
```

Comment:
We removed this code block from ValueInputPane.java as it seemed redundant. The correct functionality was observed without the explicit code block, as it's inherited. Please verify here if you remove. If there's any reason we should keep it here, we should probably add that back in ValueInputPane.java as well.

Reply:
Removed and verified.